client/src/test/java/com/paypal/selion/configuration/ProtocolConfigTest.java

```diff
@@ -0,0 +1,60 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2014 PayPal                                                                                          |
```

Comment:
This should be the year(s) the file was created and/or modified 

So it should be 2018. :)